Firebase/CoreDiagnostics/CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 # v0.1.0
-- Initial Release--for Google-use only. This library collects diagnostics and
-usage data for internal use by Firebase. Data gathered by this library will
-only be uploaded at most once every 24 hours whilst on mobile data, and more
-frequently on wifi if it's available. This library has been integrated as a
-weak dependency and can be safely removed by using a non-Cocoapods distribution
-method. You can also use the Firebase global data collection flag to opt-out of
-collecting this usage data.
\ No newline at end of file
+- Initial Release--for Google use only. This library collects diagnostics and
```

Comment:
Since this isn't a description of features added or removed, should this be not a list item at all?

Reply:
Generally I would agree, but in this case, I like the extra communication about CoreDiagnostics.

Reply:
I didn't mean remove it. I meant promote it to a paragraph.

Reply:
Done.